joomla/joomla-cms
# [4.0] Cassiopeia remove mixin a11y-visually-hidden

This PR removes a mixin that as far as I can tell just duplicated the core bootstrap mixin but was using an older, now oout of date, version.

To test

- make a copy of templates\cassiopeia\css\template.css
- apply pr
- regenerate the css (node build/build.js --compile-css)
- compare the copy and the new template.css

The only difference should be for `.container-header .mod-finder > label` which is not using the correct code.

There should be no other differences.

(I really dont understand why we have so many bootstrap overrides that dont do anything)
